code-challenges.js

```diff
@@ -8,62 +8,149 @@
+
+// const temperature1 = 42
+
+
+// const temperature2 = 350
+
 
-const temperature1 = 42
-// Expected output: "42 is below boiling point"
+// const temperature3 = 212
 
-const temperature2 = 350
-// Expected output: "350 is above boiling point"
+// const steamBubbles = () => {
+//     if(temp < steamBubbles) 
+//         return temp = "is below boiling point"
+//      else if(temp > steamBubbles)
+//         return temp = "is above boiling point"
+//      else (temp === steamBubbles)
+//         return  temp = "is at boiling point";    
+//      }
+// console.log(steamBubbles(temp))
 
-const temperature3 = 212
-// Expected output: "212 is at boiling point"
+//output SyntaxError: Unexpected identifier 'is'
+//     at internalCompileFunction (node:internal/vm:73:18)
+//     at wrapSafe (node:internal/modules/cjs/loader:1187:20)
+//     at Module._compile (node:internal/modules/cjs/loader:1231:27)
+//     at Module._extensions..js (node:internal/modules/cjs/loader:1321:10)
+//     at Module.load (node:internal/modules/cjs/loader:1125:32)
+//     at Module._load (node:internal/modules/cjs/loader:965:12)
+//     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
+//     at node:internal/main/run_main_module:23:47
+
+// Node.js v20.0.0
+// learnacademy@MacBook-Air-4 week-1-assessment-kungfuteamountain % node code-challenges.js
+// /Users/learnacademy/Desktop/week-1-assessment-kungfuteamountain/code-challenges.js:37
+//             return temperature "is below boiling point"
+//                                ^^^^^^^^^^^^^^^^^^^^^^^^
+
+// I couldnt make heads or tails of anything at this point. i may have forgotten everything after class.
+        
+// const temperature1 = 42
+// // Expected output: "42 is below boiling point"
+
+// const temperature2 = 350
+// // Expected output: "350 is above boiling point"
+
+// const temperature3 = 212
+// // Expected output: "212 is at boiling point"
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// // creat variable for both arrays then .concat() to combine
+// use .length() shoud return new and improved array's final index number after combining the arrays
+// console.log
+
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+// var worldSeriesStats= padres1984WorldSeriesRuns.push(padres1998WorldSeriesRuns)
+
+// console.log(worldSeriesStats.length)
 // Expected output: 9
+//not gettting an output.
+
+// works? outputs 10. should output 10 instead of 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
-
-const currentCohort = "Delta 2023"
+// 
+// use .revers()
+// 
+
+// const currentCohort => () { "Delta 2023"
```

Comment:
Your syntax is incorrect here, look at the syllabus and your notes to follow proper syntax for a function. Also, your pseudo code needs much more elaboration. It should go step by step in what the code is doing.